packages/node_modules/@webex/plugin-meetings/src/common/logs/logger-proxy.js

```diff
@@ -1,13 +1,14 @@
+/* eslint-disable no-unused-vars */
 import LoggerConfig from './logger-config';
 
 const LoggerProxy = {
   logger: {
-    info: () => { console.error('LoggerProxy->info#NO LOGGER DEFINED'); },
-    log: () => { console.error('LoggerProxy->log#NO LOGGER DEFINED'); },
-    error: () => { console.error('LoggerProxy->error#NO LOGGER DEFINED'); },
-    warn: () => { console.error('LoggerProxy->warn#NO LOGGER DEFINED'); },
-    trace: () => { console.error('LoggerProxy->trace#NO LOGGER DEFINED'); },
-    debug: () => { console.error('LoggerProxy->debug#NO LOGGER DEFINED'); }
+    info: (args) => { console.error('LoggerProxy->info#NO LOGGER DEFINED'); },
```

Comment:
I had to add the args because otherwise typescript was complaining when I was using the logger in the newly created ts file